scripts/spack/packages/serac/package.py

```diff
@@ -115,35 +118,44 @@ class Serac(CMakePackage, CudaPackage):
     depends_on("sundials~shared+hypre+monitoring")
 
     # Libraries that support +debug
-    debug_deps = ["mfem@4.2.0~shared+metis+superlu-dist+lapack+mpi+netcdf+sundials",
-                  "hypre@2.18.2~shared~superlu-dist+mpi"]
+    debug_deps = ["mfem@4.2.0~shared+metis+superlu-dist+lapack+mpi",
+                  "hypre@2.18.2~shared~superlu-dist+mpi~int64"]
 
     depends_on("petsc~shared", when="+petsc")
     depends_on("petsc+debug", when="+petsc+debug")
 
     for dep in debug_deps:
         depends_on("{0}".format(dep))
         depends_on("{0}+debug".format(dep), when="+debug")
-    #depends_on("mfem+netcdf", when="+netcdf")
+    depends_on("mfem+netcdf", when="+netcdf")
     depends_on("mfem+petsc", when="+petsc")
+    depends_on("mfem+sundials", when="+sundials")
+    depends_on("sundials~shared", when="+sundials")
+    depends_on("netcdf-c@4.7.4~shared", when="+netcdf")
 
     # Needs to be first due to a bug with the Spack concretizer
     depends_on("hdf5+hl@1.8.21~shared")
 
+    # Axom enables RAJA/Umpire by default
+    depends_on("axom~raja", when="~raja")
```

Comment:
Should we be toggling this based on the cuda variant instead of individually?

Reply:
I think it makes sense to keep them as a separate variant, if we start using e.g. RAJA in serac code directly we would probably want it to still be compiled when GPUs are not available, and just execute serially with RAJA in that case.